migrations/20170331095601_wilsons_as_function.js

```diff
@@ -0,0 +1,34 @@
+// http://www.evanmiller.org/how-not-to-sort-by-average-rating.html
+exports.up = function(knex, Promise) {
+  return knex.schema.raw(`
+      CREATE FUNCTION wilsons(ups int, downs int) RETURNS numeric LANGUAGE plpgsql IMMUTABLE AS $$
+      DECLARE
+        z int;
+        n int;
+        p numeric;
+        l numeric;
+        r numeric;
+        u numeric;
+      BEGIN
+        n := ups + downs;
+        IF
+          n = 0
+        THEN
+          return 0;
+        END IF;
+        z := 1.281551565545;
+        p := ups::numeric / n;
+        l := p + 1 / ( 2 * n) * z * z;
+        r := z * sqrt(p * (1 - p) / n + z * z / (4 * n * n));
+        u := 1 + 1 / n * z * z;
+        RETURN ((l - r) / u);
+      END
+      $$;
+    `);
+};
+
+exports.down = function(knex, Promise) {
+  return knex.schema.raw(`
+      DROP FUNCTION wilsons(ups int, downs int)
+    `);
+};
```

Comment:
Added as function for two primary reasons:
1) Clarity. If added inline the SQL would be too difficult to read.
2) Reuseability. The same function can be used to get the best content for admin endpoint.